app/api/orders.py

```diff
@@ -417,3 +417,64 @@ def create_paypal_payment(order_identifier):
         return jsonify(status=True, payment_id=response)
     else:
         return jsonify(status=False, error=response)
+
+# Just Testing OMISE Payment Mechanism
+@order_misc_routes.route('/orders/<string:order_identifier>/omise-checkout', methods=['POST'])
+def omise_checkout(order_identifier):
+    # TEST STUBS
+
+    print('you have reached omise transactions\n')
+    print(request.form)
+    token = request.form.get('omiseToken')
+    source = request.form.get('omiseSource')
+
+    print('\n', token, source)
+
+    omise.api_secret = 'skey_test_xxxxxxxxxxxx'
```

Comment:
@uds5501 Has this been working locally? Also, would be great if you could fetch the omise credentials from the db, similar to stripe and PayPal
```
class StripeAuthorization(SoftDeletionModel):
    """
    Stripe authorization information for an event.
    """
    __tablename__ = 'stripe_authorizations'

    id = db.Column(db.Integer, primary_key=True)
    stripe_secret_key = db.Column(db.String)
    stripe_refresh_token = db.Column(db.String)
    stripe_publishable_key = db.Column(db.String)
    stripe_user_id = db.Column(db.String)
    stripe_auth_code = db.Column(db.String)
```